src/persist-client/src/internal/metrics.rs

```diff
@@ -1112,6 +1113,10 @@ impl StateMetrics {
                 name: "mz_persist_state_writer_removed",
                 help: "count of writers removed from the state",
             )),
+            force_apply_hostname_total: registry.register(metric!(
+                name: "mz_persist_state_force_apply_hostname_total",
```

Comment:
nit: s/_total/_count/ to be consistent with the other metric names

Reply:
I'm a little torn because elsewhere we've been trying to increasingly lean on the prometheus naming convention in https://prometheus.io/docs/practices/naming/#metric-names which uses `total` for counters